spirv_reflect.c

```diff
@@ -363,13 +435,13 @@ static bool InRange(
 }
 
 static SpvReflectResult ReadU32(
-  SpvReflectPrvParser* p_parser,
+  const SpvReflectPrvParser* p_parser,
```

Comment:
Parser needs to be constant from needed call tree. Constant doesn't change any behavior.